ament_package/template/prefix_level/_local_setup_util.py

```diff
@@ -320,9 +320,9 @@ def handle_dsv_types_except_source(type_, remainder, prefix):
                 type_ == DSV_TYPE_PREPEND_NON_DUPLICATE_IF_EXISTS and
                 not os.path.exists(value)
             ):
-                comment = 'skip extending {env_name} with not existing path: ' \
-                    '{value}'.format_map(locals())
                 if _include_comments():
+                    comment = f'skip extending {env_name} with not existing ' \
+                        f'path: {value}'
```

Comment:
I believe this is now short enough to be unwrapped.

Reply:
The previous line wraps on column 80. So no, it can't be unwrapped.

Reply:
Isn't the line length 100?

https://github.com/ament/ament_package/blob/9b2bfeaa0516aa35bfc712345c5541183bd61aaa/.flake8#L4

Reply:
True, I just try to keep it under 80 characters when writing any new code / touching existing code to match what the majority of the Python community does.

Reply:
Why? I mean we're not planning on changing our line length, that would be really disruptive. I don't see any value in doing that, since the point of the longer line length limit is so that we can avoid wraps like this one...

You can do what you want I guess, but I think it should be unwrapped personally.